pychunkedgraph/backend/history.py

```diff
@@ -479,9 +473,7 @@ def get_all_log_entries(cg_instance):
     log_rows = cg_instance.read_log_rows()
     for operation_id in range(cg_instance.get_max_operation_id()):
         try:
-            log_entries.append(
-                LogEntry(log_rows[operation_id], log_rows[operation_id]["timestamp"])
-            )
+            log_entries.append(LogEntry(log_rows[operation_id]))
```

Comment:
why did you remove the timestamp here?

Reply:
The class does not have a parameter for timestamp. There is inconsistency in `read_log_row` and `read_log_rows` return values, single log returns a tuple with log dict and a timestamp but in log rows dict has a `timestamp` key.
I think currently `LogEntry` class is only using return value of `read_log_rows`.

Reply:
okay, @kaikue making you aware of this as I think your endpoint for the leaderboard is the main user of this

Reply:
I don't think I use this function (it gets called by `common.py:change_log` while I'm using `common.py:tabular_change_log_recent`), but I will keep this in mind if anything weird happens